icicle/tests/test_hash_api.cpp

```diff
@@ -1067,3 +1067,51 @@ TEST_F(HashApiTest, poseidon_tree)
 }
 
 #endif // POSEIDON
+
+/*======================= (TODO) Move to Sumcheck test-suite =======================*/
```

Comment:
why didn't you open a new file for that?

Reply:
I should do it yes. We should add a SUMCHECK feature to cmake right? and build the test accordingly